tests/test_application/test_application.py

```diff
@@ -43,6 +61,8 @@ def test_application_contents_with_unexpected_data(flask_test_client):
     assert response.status_code == 400, "Unexpected status code"
     assert b"Incorrect or missing APPLICATION data" in response.data
 
+    assert response.status_code == 400
```

Comment:
Is this a duplicate of line 61?

Reply:
Mistake when sorting out conflicts 😄 
Thanks for pointing it out